src/components/CreateSecret/ServiceAccountSelector.js

```diff
@@ -118,6 +109,7 @@ const ServiceAccountSelector = props => {
           },
           { kind: 'Secrets', selectedNamespace: namespace }
         )}
+        toolbarButtons={[{ onClick: handleClose, text: 'Finish' }]}
```

Comment:
need to move the `intl.formatMessage` here too

Reply:
Done that now 😸